qiskit/pulse/macros.py

```diff
@@ -92,6 +163,63 @@ def measure(
     return schedule
 
 
+def _measure_v2(
+    qubits: List[int],
+    target: Target,
+    meas_map: Union[List[List[int]], Dict[int, List[int]]],
+    qubit_mem_slots: Dict[int, int],
+    measure_name: str = "measure",
+) -> Schedule:
+    """Return a schedule which measures the requested qubits according to the given
+    target and measure map, or by using the defaults provided by the backendV2.
+
+    Args:
+        qubits: List of qubits to be measured.
+        backend (Union[Backend, BaseBackend]): A backend instance, which contains
+            hardware-specific data required for scheduling.
```

Comment:
```suggestion
```
This doesn't exist.

Reply:
I fixed at https://github.com/Qiskit/qiskit-terra/pull/9987/commits/402984987df12b4fc46459db1a57bf8a80d35486.